plugins/hydra_nevergrad_sweeper/tests/test_nevergrad_sweeper_plugin.py

```diff
@@ -84,8 +140,8 @@ def test_nevergrad_example(with_commandline: bool, tmpdir: Path) -> None:
         cmd += [
             "db=mnist,cifar",
             "batch_size=4,8,12,16",
-            "lr=log:0.001:1.0",
-            "dropout=0:1",
+            "lr=tag(log, interval(0.001, 1.0))",
+            "dropout=interval(0,1)",
         ]
     subprocess.check_call(cmd)
     returns = OmegaConf.load(f"{tmpdir}/optimization_results.yaml")
```

Comment:
can you switch to using `get_run_output()` for this and other tests here?

Reply:
done! thx